src/modules/functions.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable no-restricted-globals */
+// add button function
+let listTasks = [];
+if (localStorage.getItem('listTasks') !== null) {
+  listTasks = JSON.parse(localStorage.getItem('listTasks'));
+}
+// DOM manipulation
+const addbtn = document.querySelector('.add');
+const msg = document.querySelector('.msg');
+const txtField = document.getElementById('taskTextField');
+const taskContainerFromhtml = document.querySelector('.tasks');
+let removeButton = [];
+
+function add(arr) {
+  if (localStorage.getItem('listTasks') !== null) {
+    arr = JSON.parse(localStorage.getItem('listTasks'));
+  }
+  arr.push(txtField.value);
+  localStorage.setItem('listTasks', JSON.stringify(arr));
```

Comment:
- [ ] Please store the objects as per the requirement of this milestone, you only stored the text value, but we should also show the index property starting from 1, 2, 3... and the completed property which should have boolean values. In the next capstone, you are also going to implement the checkbox functionality which will update the objects in the local storage.